core/io/resource.cpp

```diff
@@ -708,6 +716,20 @@ String Resource::get_id_for_path(const String &p_referrer_path) const {
 #endif
 }
 
+void Resource::_validate_property(PropertyInfo &p_property) const {
+	if (p_property.name == "editor_description") {
+		if (is_built_in()) {
+			// For sub resources, we store editor description directly in the property map.
+			// As we don't need to show the tooltip, this simplifies implementation.
+			p_property.usage |= PROPERTY_USAGE_STORAGE;
+			// Therefore, every resource that can be stored as a sub resource automatically supports editor descriptions,
+			// as they will always be serialized with ResourceSaverText or ResourceSaverBinary, depending on the scene or resource owning them.
+		} else if (!ResourceLoader::has_editor_description_support(get_path())) {
+			p_property.usage |= PROPERTY_USAGE_READ_ONLY;
+		}
```

Comment:
Should we hide the property entirely instead? It's always going to be empty if the format doesn't support it, so I don't see much point to having a read-only field here.